mergify_cli/ci_issues/cli.py

```diff
@@ -0,0 +1,82 @@
+import click
+
+from mergify_cli import utils
+from mergify_cli.ci_issues import junit_upload as junit_upload_mod
+
+
+ci_issues = click.Group(
+    "ci-issues",
+    help="Interact with Mergify's CI Issues",
+)
+
+
+@ci_issues.command(help="Upload JUnit XML reports")
+@click.option(
+    "--api-url",
+    "-u",
+    help="URL of the Mergify API",
+    required=True,
+    envvar="MERGIFY_API_SERVER",
+    default="https://api.mergify.com/v1",
```

Comment:
```suggestion
    default="https://api.mergify.com/",
```
I feel that should be the root server here, cc @sileht

Reply:
Yes